game.py

```diff
@@ -47,20 +47,27 @@ class AppleFinder(GridGame):
     PIT = HOLE
     APPLE = chr(15)
     APPLE = 'O'
+    BASE = chr(233)
 
     def __init__(self, random):
         self.random = random
         self.running = True
         self.in_pit = False
-        centerx = self.MAP_WIDTH // 2
-        centery = self.MAP_HEIGHT // 2
-        self.player_pos = [centerx, centery]
+
+
+        # place robot and base randomly
+        x = self.random.randint(0, self.MAP_WIDTH - 1)
+        y = self.random.randint(0, self.MAP_HEIGHT - 1)
+        self.base_bos = [x, y]
```

Comment:
Spelling error?

Reply:
I edited the file in the commit through the web interface, should be g2g.